roles/system/tasks/main.yml

```diff
@@ -121,15 +100,10 @@
    - sqlite3
    - elinks
    - mosh
-  tags: ['master','custom','update']
-
-- name: Install avahi packages
```

Comment:
This task has to stay this way as installing avahi package in a chroot (this is the case when building koombook image) lead to some issue. 
Those packages than have to be installed during custom run only